src/classes/issues-processor.ts

```diff
@@ -911,6 +922,48 @@ export class IssuesProcessor {
     return this.options.removeStaleWhenUpdated;
   }
 
+  private async _removeLabelsWhenUpdatedFromStale(
+    issue: Issue,
+    labels: Readonly<string>[]
+  ): Promise<void> {
+    if (!labels.length) {
+      return;
+    }
+
+    const issueLogger: IssueLogger = new IssueLogger(issue);
+
+    issueLogger.info(
+      `Removing any labels specified via remove-labels-when-updated-from-stale...`
+    );
+
+    for (const label of labels.values()) {
+      await this._removeLabel(issue, label);
```

Comment:
Unless I'm missing something, it doesn't look like there's a single method for that: https://docs.github.com/en/rest/reference/issues#labels

Reply:
Indeed.
In that case, it would be nice I think to explicitly mention in the readme when describing the option that it will consume one operation per label removal, and it's to use carefully. WDYT?